src/app/screens/ConfirmPayment/index.jsx

```diff
@@ -14,7 +14,7 @@ class ConfirmPayment extends React.Component {
     super(props);
     this.history = createHashHistory();
     this.state = {
-      budget: (this.props.invoice?.valueSat || 0) * 10,
+      budget: this.props.invoice?.mtokens || 0,
```

Comment:
```suggestion
      budget: (this.props.invoice?.tokens || 0) * 10,
```
was it intentional to use mtokens here? when I initially wrote the code I just thought a budget of 10 times the invoice amount sounds good...no big reasoning behind.

Reply:
Nope, should me tokens in this case indeed!